onnxscript/ir/passes/common/constant_manipulation.py

```diff
@@ -23,11 +23,14 @@ class LiftConstantsToInitializersPass(ir.passes.InPlacePass):
     Attributes:
         lift_all_constants: Whether to lift all Constant nodes, including those that does not contain a tensor attribute (e.g. with value_ints etc.)
             Default to False, where only Constants with the ``value`` attribute are lifted.
+        size_limit: The minimum size of the tensor to be lifted. If the tensor contains
+            less than this number of elements, it will not be lifted.
     """
 
-    def __init__(self, lift_all_constants: bool = False):
+    def __init__(self, lift_all_constants: bool = False, size_limit: int = 16):
         super().__init__()
```

Comment:
How do we determine what an appropriate default size_limit value will be?

Reply:
Just empirically. We don't want constants that represents shapes, axes etc. to be lifted